src/store/create-level.ts

```diff
@@ -1,11 +1,10 @@
 import type { AbstractDatabaseOptions, AbstractLevel } from 'abstract-level';
 
-import { Level } from 'level';
-
 export type LevelDatabase<K, V> = AbstractLevel<string | Buffer | Uint8Array, K, V>;
 
 export type LevelDatabaseOptions<K, V> = AbstractDatabaseOptions<K, V>;
 
-export function createLevelDatabase<K, V>(location: string, options?: LevelDatabaseOptions<K, V>): LevelDatabase<K, V> {
+export async function createLevelDatabase<K, V>(location: string, options?: LevelDatabaseOptions<K, V>): Promise<LevelDatabase<K, V>> {
+  const { Level } = await import('level');
```

Comment:
Can you add the summary you had on this PR as comment, feels like important context not trivial to reader of the code.

Reply:
should `level` be a peer dep to really cement this boundary and prevent it from slipping back in to the codebase?

Reply:
what's your understanding of peer dependency @shamilovtim ? i just realized i didn't really know _what_ exactly the distinction meant. after reading [this](https://nodejs.org/ko/blog/npm/peer-dependencies/#the-solution-peer-dependencies)

> Some way of saying, "I only work when plugged in to version 1.2.x of my host package, so if you install me, be sure that it's alongside a compatible host." We call this relationship a peer dependency.

i feel like making `level` a peer dependency is the opposite of what we want. but maybe i'm not understanding `peerDependencies` correctly